install.sh

```diff
@@ -104,18 +104,29 @@ Standalone release has been installed into $STANDALONE_INSTALL_PREFIX/bin/$STAND
 
 EOF
 
-	if [ "$STANDALONE_INSTALL_PREFIX" != /usr/local ]; then
+	CODER_COMMAND="$(command -v "$STANDALONE_BINARY_NAME")"
+
+	if [ ! "$CODER_COMMAND" ]; then
 		cath <<EOF
 Extend your path to use Coder:
-PATH="$STANDALONE_INSTALL_PREFIX/bin:\$PATH"
+
+  $ PATH="$STANDALONE_INSTALL_PREFIX/bin:\$PATH"
 
 EOF
-	fi
-	cath <<EOF
-Run Coder:
-  $STANDALONE_BINARY_NAME server
+	elif [ "$CODER_COMMAND" != "$STANDALONE_BINARY_LOCATION" ]; then
+		echo_path_conflict "$CODER_COMMAND" "$STANDALONE_INSTALL_PREFIX"
```

Comment:
Do we fail if there is a path conflict?

Reply:
No. :\ This is just detecting the conflict after install, which isn't ideal, but detecting the conflict before installing is a lot harder.

Reply:
I tried to make the documentation as helpful as possible to make up for it, with the direct link so that people will (hopefully) actually read it